packages/xstate-inspect/test/inspect.test.ts

```diff
@@ -1,12 +1,14 @@
+import { JSDOM } from 'jsdom';
 import { assign, createMachine, interpret } from 'xstate';
 import { createDevTools, inspect } from '../src';
 
-afterEach(() => {
-  // this clears timers, removes global listeners etc
-  // I'm not sure if this is 100% safe to do
-  // it's not clear if the window object after this operation is still usable in the same way (is it recyclable?)
-  // it does seem to cover our needs so far though
-  window.close();
+const createNewWindow = () => new JSDOM('').window;
+
+beforeEach(() => {
+  // @ts-expect-error
+  global.window = createNewWindow();
+  // @ts-expect-error
+  window.open = jest.fn(createNewWindow);
```

Comment:
`jest` doesn't clean up the `jsdom` environment for each test (anymore? I guess), so `window.close()` was breaking everything. 

Instead we create a new `JSDOM()` instance and change the global `window` to that. 

Also mocked `window.open` to return a new `window` instance since that functionality appears to have been removed at some point.